app/services/deploy_service.rb

```diff
@@ -32,5 +37,12 @@ def self.create_github_pull_request(deploy_strategy)
       # PR already exists
     end    
   end
+
+  def self.post_slack_notification()
+    slack = Slack::Web::Client.new
+    slack.chat_postMessage(channel: '#dev', message: "<#{pr.html_url}|Deploy PR> ready for #{repo}", as_user: true)
+  rescue StandardError
+    # ok if we don't notify slack
+  end
```

Comment:
Just a note here:

When we enable automatic deployments, if (for whatever reason) the slack message fails to send then we should definitely _not_ queue up the automated merge, ha. That would be... unfortunate.